zebra-chain/src/orchard/tree.rs

```diff
@@ -302,6 +289,51 @@ pub struct NoteCommitmentTree {
     cached_root: std::sync::RwLock<Option<Root>>,
 }
 
+impl ZcashSerialize for Frontier<Node, MERKLE_DEPTH> {
+    fn zcash_serialize<W: io::Write>(&self, mut writer: W) -> Result<(), io::Error> {
+        //
+        merkle_tree::write_frontier_v1(&mut writer, self)
```

Comment:
You can use:
```rust
let mut data = Vec::new();
merkle_tree::write_frontier_v1(&mut data, tree);
```

Because `Vec` implements `Write`:
https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-Write-for-Vec%3Cu8,+A%3E